common/src/services/cipher.service.ts

```diff
@@ -68,12 +71,15 @@ export class CipherService implements CipherServiceAbstraction {
     // tslint:disable-next-line
     _decryptedCipherCache: CipherView[];
 
+    private workerThreshold: number = 250;
```

Comment:
Seems like this should be a const outside the class like Keys, DomainMatchBlacklist, etc